pecan/tests/test_util.py

```diff
@@ -93,3 +93,23 @@ def wrapped(*a, **kw):
         actual = util.getargspec(dec(True)(
             self.controller.static_index))
         assert expected == actual
+
+    def test_nested_cells(self):
+
+        def before(handler):
+            def deco(f):
```

Comment:
`f` is the *actual* underlying function with the argspec we care about; it has a `self` argument, so it's probably an instance method